index.html

```diff
@@ -0,0 +1,204 @@
+        </p>
+        <ul class="languages">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </main>
+      <main class="Works_main">
+        <img class="big_pictures" src="image/picture_4.png" alt="" />
+        <h2>
+          Multi-Post<br />
+          Stories
+        </h2>
+        <ul class="Works_Tony">
+          <li>CANOPY</li>
+          <li><img src="image/Card_1.png" alt="" /></li>
+          <li>Back End Dev</li>
+          <li><img src="image/Card_1.png" alt="" /></li>
+          <li>2015</li>
+        </ul>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </main>
+    </section>
+    <div class="borderabout">
+      <section>
+        <div class="Body_content_title">
+          <h3>
+            About<br />
+            Myself
+          </h3>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <h4>LET’S CONNECT</h4>
+          <ul>
+            <li><img src="image/Twittwer_vector.png" alt="" /></li>
+            <li><img src="image/Linkedin_icon.png" alt="" /></li>
+            <li><img src="image/Pieace_vector.png" alt="" /></li>
+            <li><img src="image/M_vector.png" alt="" /></li>
+            <li><img src="image/Github_vector.png" alt="" /></li>
+          </ul>
+          <button class="btn">Get my resume</button>
+        </div>
+        <div class="fram">
+          <ul>
+            <li>
+              <h4>Languages</h4>
+              <img class="chevron" src="image/chevron-down-solid.svg" alt="" />
+            </li>
+            <div>
+              <img class="elipse" src="image/Ellipse_JS.png" alt="" />
+              <h5>javascript</h5>
+            </div>
+            <div>
+              <img class="elipse" src="image/Ellipse_HTML.png" alt="" />
+              <h5>HTML</h5>
+            </div>
+            <div>
+              <img class="elipse" src="image/Ellipse_CSS.png" alt="" />
+              <h5>CSS</h5>
+            </div>
+            <li>
+              <h4>Frameworks</h4>
+              <img class="chevron" src="image/chevron-right-solid.svg" alt="" />
+            </li>
+            <hr />
+            <li>
+              <h4>Skills</h4>
+              <img
+                class="chevron"
+                src="images/chevron-right-solid.svg"
+                alt=""
+              />
+            </li>
+            <hr />
+          </ul>
+        </div>
+      </section>
```

Comment:
These are the following issues that I found in your design:
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219760167-8125a814-934c-4352-a893-02226a2f3e74.png) | ![image](https://user-images.githubusercontent.com/61361037/219760311-b0ce506c-912f-49cd-bbb4-02d4b3a68bd6.png) |

1. The size of `hx` is missing the styling. 
2. LETS CONNECT misses the `font-family` and `color` which is `#7F8CFF`
3. Button have the same styling issue as discussed above.
4. Languages missing font styling
5. Missing right arrow icon